plugins/brimcap/brimcap-plugin.ts

```diff
@@ -177,10 +177,10 @@ export default class BrimcapPlugin {
     // )
 
     const tsString = ts.toString()
-    const dur = log.get("duration") as zed.Duration
+    const dur = log.try("duration") as zed.Duration
```

Comment:
Try can return null. In the returned object, check for null before calling `isSet()`.

Reply:
I ended up doing just that down a bit further at https://github.com/brimdata/brim/pull/1617/files#diff-00e37cfd83eb8f0d73899b431f6d59bbad7c73d7bc1e8ea6acbf1eba935ee120R189

Reply:
Oh wait I misunderstood, disregard